### PR TITLE
Added documentation for environments.

### PR DESCRIPTION
**Pull request by ppasupat**
# Description

- Created a class for each environment and wrote documentation in docstrings.
- Generated Markdowns by adapting [`gen_md.py`](https://github.com/Farama-Foundation/Gymnasium/blob/main/docs/scripts/gen_mds.py) from Gymnasium.

This PR depends on PR #21.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `pre-commit run --all-files` (see `CONTRIBUTING.md` instructions to set it up)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: miniwob/scripts/gen_env_classes.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    None
+    \"\"\"
+
+    subdomain = "{name}"
+"""
+
+
+def _raw_env_name(env_id: str) -> str:
+    """Convert ID like miniwob/click-test-v1 to click-test."""
+    match = re.search("^miniwob/(.*)-v[0-9.]*$", env_id)
```
Comment (pseudo-rnd-thoughts) [inline miniwob/scripts/gen_env_classes.py:35]: Optional, you can use `gymnasium.envs.registration.parse_env_id` which will return the namespace, env_name and version then you just need to use the env_name variable

**Review by pseudo-rnd-thoughts — APPROVED** [review]
Looks good

To keep the documentation update, normally, the documentation is generated on each PR. 
@mgoulao Can we add this feature like we have in Gymnasium?

**Comment by mgoulao** [comment]
@ppasupat add a workflow like this, you need to change some parts, like "Install Gymnasium" or "Install dependencies" [https://github.com/Farama-Foundation/Gymnasium/blob/main/.github/workflows/build-docs.yml](https://github.com/Farama-Foundation/Gymnasium/blob/main/.github/workflows/build-docs.yml).
Generated files should be ignored by git, so I would suggest you remove all Markdown files that are generated using `gen_md.py` and add a line to the `.gitignore` with something like `docs/environments/miniwob/*.md`.

File: miniwob/scripts/gen_env_classes.py (hunk 1 above)
Comment (ppasupat) [reply miniwob/scripts/gen_env_classes.py]: Done.

**Comment by ppasupat** [comment]
I added `build-docs.yml` and the files it mentions. I also added `.gitignore`.

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+# -- Project information -----------------------------------------------------
+from typing import Any, Dict
+
+project = "MiniWoB++"
+copyright = "2023"
+author = "Farama Foundation"
+
+# The full version, including alpha/beta/rc tags
+release = "0.0.1"
```
Comment (pseudo-rnd-thoughts) [inline docs/conf.py:24]: This needs to be dynamically set depending on the miniwob version
https://github.com/Farama-Foundation/Gymnasium/blob/f6d41e85f9965fbd2258a8b34855ae434bb70b75/docs/conf.py#L32

**Pull request merged**